src/Tizen.Network.Nsd/Tizen.Network.Nsd/DnssdService.cs

```diff
@@ -218,14 +220,33 @@ public IPAddressInformation IP
             }
         }
 
-        private void GetTxtRecord(out ushort length, out byte[] value)
+        /// <summary>
+        /// Returns all raw TXT records.
+        /// </summary>
+        /// <remarks>
+        /// After adding the TXT record call for get TXT record.
+        /// </remarks>
+        /// <returns>Returns empty bytes in case of no records available else returns raw text tecords.</returns>
```

Comment:
I suspect TXT is not exactly the same as "text".
```suggestion
        /// <returns>Returns empty bytes array in case TXT record has not been set, else returns raw TXT record.</returns>
```

Reply:
Fixed